app/design/adminhtml/default/default/template/downloadable/product/composite/fieldset/downloadable.phtml

```diff
@@ -30,7 +30,11 @@
         <dl>
         <?php $_links = $this->getLinks(); ?>
         <?php $_isRequired = $this->getLinkSelectionRequired(); ?>
-            <dt><label<?php if ($_isRequired) echo ' class="required"' ?>><?php if ($_isRequired) echo '<em>*</em>' ?><?php echo $this->escapeHtml($this->getLinksTitle()); ?></label></dt>
+            <dt><label<?php if ($_isRequired) {
```

Comment:
this is less readable. Maybe we should have different phpcs rules for templates?
In such cases I would refactor it to:

```
<dt><label<?= $_isRequired ? ' class="required"' : '' ?>>
```
but it's another discussion.

Reply:
> Maybe we should have different phpcs rules for templates?

Tried to avoid it.

Reply:
I've to agree with @tmotyl

Reply:
okay, i'll set it to draft. maybe someone else want to fix it. (no worries, but i'll not spent much time to fix template files)